workshop/exercise-8/README.md

```diff
@@ -0,0 +1,57 @@
+## A/B Testing with knctl
+
+Maybe we want to slowly roll users over from our old version to the new version, or do some A/B testing of the new version. We can use the knctl rollout command to route traffic percentages to our revisions.
+
+1. Check the current route percentages:
+
+	```
+	knctl route list
+	```
+
+	Output:
+	```
+	Routes in namespace 'default'
+
+	Name         Domain                                                              Traffic                   Annotations  Conditions  Age  
+	fib-knative  fib-knative.default.bmv-knative.us-east.containers.appdomain.cloud  100% -> fib-knative  -            3 OK / 3    20h  
+
+	1 routes
+
+	Succeeded
+	```
+
+2. Send 50% of the traffic to the latest revision, and 50% to the previous revision. Notice that we're using the previous and latest tags that were created for us as a part of the knctl deploy command.
+
+	```
+	knctl rollout --route fib-knative -p fib-knative:latest=50% -p fib-knative:previous=50%
+	```
+
+3. Check the route percentages to confirm they've updated:
+
+	```
+	knctl route list
+	```
+
+	Output:
+	```
+	Routes in namespace 'default'
+
+	Name         Domain                                                              Traffic                   Annotations  Conditions  Age  
+	fib-knative  fib-knative.default.bmv-knative.us-east.containers.appdomain.cloud  50% -> fib-knative-00003  -            3 OK / 3    5m  
+																					 50% -> fib-knative-00002                          
```

Comment:
the formatting on this line is still of - the "50% -> fib-knative-0002" is way off the side of the screen.

Reply:
ugh, i think github markdown is slightly different than the .md viewer I use.  Let me mess with this again

Reply:
okay, updated.